src/events/components/ImageView/index.tsx

```diff
@@ -83,36 +81,9 @@ export const ImageView = ({  }: IProps) => {
   return (
     <>
       <div className={style.eventGrid}>
-        <div className={style.eventColumn}>
-          {displayEvents.eventsLeft[0] ? (
-            <>
-              <LargeEvent {...displayEvents.eventsLeft[0]} />
-              <SmallEventColumn events={displayEvents.eventsLeft.slice(1, 4)} />
-            </>
-          ) : (
-            <LargeEventPlaceholder event_type={2} />
-          )}
-        </div>
-        <div className={style.eventColumn}>
-          {displayEvents.eventsMiddle[0] ? (
-            <>
-              <LargeEvent {...displayEvents.eventsMiddle[0]} />
-              <SmallEventColumn events={displayEvents.eventsMiddle.slice(1, 4)} />
-            </>
-          ) : (
-            <LargeEventPlaceholder event_type={3} />
-          )}
-        </div>
-        <div className={style.eventColumn}>
-          {displayEvents.eventsRight[0] ? (
-            <>
-              <LargeEvent {...displayEvents.eventsRight[0]} />
-              <SmallEventColumn events={displayEvents.eventsRight.slice(1, 4)} />
-            </>
-          ) : (
-            <LargeEventPlaceholder event_type={1} />
-          )}
-        </div>
+        <EventColumn events={displayEvents.eventsLeft} event_type={2} />
+        <EventColumn events={displayEvents.eventsMiddle} event_type={3} />
+        <EventColumn events={displayEvents.eventsRight} event_type={1} />
```

Comment:
Also the numbers here should rather be `EventTypeEnum.BEDPRES`, `EventTypeEnum.KURS` and `EventTypeEnum.SOSIALT`

Reply:
^agree